beta-prompts/beta_prompts/visualizations.py

```
@@ -0,0 +1,1233 @@
+
+        for i in future_weeks:
+            # Base trend with some acceleration
+            base_prediction = historical_performance[-1] + 0.015 * i
+            # Add uncertainty that increases with time
+            uncertainty = 0.02 * np.sqrt(i)
+
+            predicted_performance.append(base_prediction)
+            uncertainty_upper.append(base_prediction + uncertainty)
+            uncertainty_lower.append(base_prediction - uncertainty)
+
+        all_weeks = weeks + future_weeks
```

Ruff (0.12.2)

Local variable `all_weeks` is assigned to but never used

(F841)